### PR TITLE
dird: `list jobs`: add `level` keyword and accept a list of job levels

### PR DESCRIPTION
**Pull request by alaaeddineelamri**
#### Description

This PR add the `level` keyword that works as an alias to `joblevel`, and also enables the `list jobs` command to parse a list of job levels (i.e. `list jobs level=F,I,D`)

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)
- [Boy Scout Rule](https://docs.bareos.org/DeveloperGuide/generaldevel.html#boy-scout-rule)

### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)
Make sure you check/merge the PR using `devtools/pr-tool` to have some simple automated checks run and a proper changelog record added.

##### General
- [x] Is the PR title usable as CHANGELOG entry?
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted
~~Check backport line~~
- [x] Required backport PRs have been created

##### Source code quality
- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR

##### Tests
- [x] Decision taken that a test is required (if not, then remove this paragraph)
- [x] The choice of the type of test (unit test or systemtest) is reasonable
- [x] Testname matches exactly what is being tested
- [x] On a fail, output of the test leads quickly to the origin of the fault

### CODE REVIEW

File: systemtests/tests/python-bareos/test_list_command.py [hunk 1]
```diff
@@ -258,15 +268,49 @@ def test_list_jobs(self):
...
+        result = director.call("list jobs joblevel=F")
+        self.assertTrue(result["jobs"])
         for job in result["jobs"]:
-            self.assertTrue(job["type"] == "B" or job["type"] == "R")
+            self.assertEqual(job["level"], "F")
+
+        result = director.call("list jobs level=I")
+        self.assertTrue(result["jobs"])
+        for job in result["jobs"]:
+            self.assertEqual(job["level"], "I")
```
Comment (sebsura) [inline systemtests/tests/python-bareos/test_list_command.py:299]: I would also add this additional test:
```
result2 = director.call("list jobs joblevel=I")
self.assertTrue(result2["jobs"])
self.assertTrue(result == result2)
```
To ensure that both queries result in the same thing.

File: systemtests/tests/python-bareos/test_list_command.py [hunk 2]
```diff
@@ -258,15 +268,49 @@ def test_list_jobs(self):
         for job in result["jobs"]:
             self.assertEqual(job["type"], "B")
 
-        result = director.call("list jobs jobtype=Backup")
+        result = director.call("list jobs jobtype=Restore")
```
Comment (sebsura) [inline systemtests/tests/python-bareos/test_list_command.py:271]: Im not sure I understand this change.  Why did this happen ?

**Review by sebsura — CHANGES_REQUESTED** [review]
Good work.  I left some suggestions.

File: systemtests/tests/python-bareos/test_list_command.py (hunk 2 above)
Comment (alaaeddineelamri) [reply systemtests/tests/python-bareos/test_list_command.py:271]: to make sure jobtype `Restore` would actually return something. I tested the backup in the previous one

It tests two things at once: that the full status name gets correctly parsed and interpreted, and also that the restore is correctly listed.

**Review by sebsura — APPROVED** [review]
Looks good.

**Pull request merged**